account_payment_return_import_iso20022/wizard/camt_parser.py

```diff
@@ -40,7 +40,7 @@ def add_value_from_node(ns, node, xpath_str, obj, key, join_str=None):
         If xpath_str is a list (or iterable), it will be seen as a series
         of search path's in order of preference. The first item that results
         in a found node will be used to set a value in the dictionary."""
-        if not isinstance(xpath_str, (list, tuple)):
+        if not isinstance(xpath_str, list | tuple):
```

Comment:
Why this change? It was working before and AFAIK, this hasn't changed across Python versions.

Reply:
If not changed, pre-commit fails.

Check error message:

account_payment_return_import_iso20022/wizard/camt_parser.py:43:16: UP038 Use `X | Y` in `isinstance` call instead of `(X, Y)`

Reply:
Oh, I didn't know. It seems then a legacy notation that is still compatible...